src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -475,7 +475,7 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
 {1} - Project path</comment>
   </data>
   <data name="Error_NoProjectFound" xml:space="preserve">
-    <value>Error while adding package '{0}' to project '{1}'. dotnet add package command supports adding package to one project at a time. But no project was found at the project path '{0}'.</value>
+    <value>Error while adding package '{0}' to project '{1}'. dotnet add package command supports adding package to one project at a time. But no project was found at the project path '{1}'.</value>
```

Comment:
adding **a** package

or just remove the adding a package part, this should be obvious.